assets/config/0.5.6-coins.json

```diff
@@ -5228,6 +5228,29 @@
     "active": false,
     "currently_enabled": false
   },
+  "AVN": {
+    "coin": "AVN",
+    "name": "Avian",
+    "coinpaprika_id": "avn-avian",
+    "coingecko_id": "avian-network",
+    "nomics_id": "avn3-avian",
```

Comment:
id is `AVN3` as in `data-base="AVN3"` from https://nomics.com/assets/avn3-avian/widget

Reply:
Thanks, I have updated this to `AVN3`.